src/util/index.js

```diff
@@ -33,6 +36,47 @@ export function buildContent(options : ContentConfig) : string {
     return content || '';
 }
 
+export function getBuyerCountry(fundingOption : string) : string {
+    switch (fundingOption) {
+    case 'PayUponInvoice': {
+        return 'DE';
+    }
+    case 'boletobancario': {
+        return 'BR';
+    }
+    default: {
+        return 'US';
+    }
+    }
+}
+
+/**
+ * This function defaults the legal text language local based on
+ * payment option and overrides the user preferences
+ * @param {*} fundingOption
+ * @returns
+ */
+export function getLegalLocale(fundingOption : string) : Object  {
```

Comment:
why do we want to overwrite the user preference? then, user may see the checkout content in one locale, and the legal statement in a different locale. also, it is hard to determine the locale by country, since many countries have multiple locales.

Reply:
It's the other way @weihou , The User Preference overrides the default language settings for the Payment Option. We need a default Local Language incase the `getLocale()` doesn't return anything

Reply:
I suggest

1. call getLocale() and getBuyerCountryFromFundingSource
2. compare the country returned from these 2 calls
3. if the country is the same, use the locale from getLocale()
4. if countries are different, set the locale to `en-<country from getBuyerCountryFromFundingSource.`